components/homepage/hero/Hero.tsx

```diff
@@ -11,9 +11,11 @@ import {
 import NextLink from 'next/link'
 import { useAppContext } from '@store/hooks'
 import { PROTOCOL, REMOTE_HOSTNAME } from '@helpers/config'
+import { useWebDetails } from '@hooks/webs'
 
 export default function Hero() {
   const { isMobile } = useAppContext()
+  const { webDetails } = useWebDetails()

```

Comment:
I would expect `webDetails` to represent the details for an individual web. There is already a `useWebs` and an associated API route though, do you think we can make use of those?

Reply:
There was already a function with this name, though I could just rename it, the useWeb hook could be useFullWeb as it includes listings

Reply:
Good point - I'd be happy for you to change `useWebs` to `useFullWebs`. Or let me know if you don't have time and I can continue this.

Reply:
I ended up going with useWebs for the Locations and useWebListings for the locations with some listing information.